INN/umbrella-inndev
# Don't set limit on amount of posts displayed on Amplify feed

## Changes

This pull request makes the following changes:

- Adds `'posts_per_page' => -1,` to the feed query args so all posts are shown

## Why



For #140/#127

## Testing/Questions

Features that this PR affects:

- Amplify feed query


Questions that need to be answered before merging:


Steps to test this PR:

1. Add more than 10 posts to your saved tags list. Make sure all display on the amplify feed.
